terraform/aws/vpc/main.tf

```diff
@@ -0,0 +1,22 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "2.15.0"
+
+  name               = "${var.cluster_name}-vpc"
+  cidr               = "10.18.0.0/16"
+  azs                = ["${var.region}a"]
+  enable_nat_gateway = true
+  enable_vpn_gateway = true
+
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+
+  public_subnets = ["10.18.128.0/20"]
+  tags = {
+    Terraform   = "true"
```

Comment:
Please, run `terraform fmt`, it fix many style guide errors

Reply:
done